opal/mca/pmix/pmix3x/pmix3x_client.c

```diff
@@ -99,6 +99,11 @@ int pmix3x_client_init(opal_list_t *ilist)
         ninfo = 0;
     }
 
+    /* check for direct modex use-case */
+    if (opal_pmix_base_async_modex && !opal_pmix_collect_all_data) {
+        opal_setenv("PMIX_MCA_gds", "hash", true, &environ);
```

Comment:
@rhc54 
This line disables disables `dstore`. We accidentally found that. 
Can you remind me why this was done?
Also I think this kind of change should come in a separate PR as it significantly changes the behavior yet quite small to find it between other 400+ changed lines.

@jladd-mlnx FYI

Reply:
@karasevb

Reply:
@rhc54 
I see that this PR was filed on Nov, 7.
And we had a dix for the dstore and local commits on
Dec, 20:
https://github.com/pmix/pmix/pull/613 

Am I right that it supposed to fix this issue and we can remove this?

Reply:
@rhc54 FYI we discovered the issue with pmix/pmix#613.
@karasevb is going to PR the fix today

Reply:
PR: https://github.com/pmix/pmix/pull/697